src/input/mpegts/satip/satip.c

```diff
@@ -384,12 +398,6 @@ satip_device_hack( satip_device_t *sd )
     sd->sd_pids_max    = 128;
     sd->sd_pids_len    = 2048;
     sd->sd_no_univ_lnb = 1;
-  } else if (strstr(sd->sd_info.manufacturer, "AVM Berlin") &&
```

Comment:
Okay, I moved the code back to hack function and added a new option called "Disable FRITZ!-specific workarounds" -- if it's not set the hacks are forced on, if it is set the hack is ignored and user's own settings will prevail.